packages/jss-plugin-default-unit/src/defaultUnits.js

```diff
@@ -2,9 +2,9 @@
 
 import {hasCSSTOMSupport} from 'jss'
 
-const px = hasCSSTOMSupport ? window.CSS.px : 'px'
-const ms = hasCSSTOMSupport ? window.CSS.ms : 'ms'
-const percent = hasCSSTOMSupport ? window.CSS.percent : '%'
+const px = hasCSSTOMSupport && CSS ? CSS.px : 'px'
```

Comment:
You need to make sure CSS is defined or check with window.CSS
![image](https://user-images.githubusercontent.com/5635476/62971695-19ef0500-be1b-11e9-9a00-4f460cd26339.png)

Reply:
hasCSSTOMSupport did that check already, I am just making flow happy